db/migrate/20160501143232_add_badge_status_to_projects.rb

```diff
@@ -0,0 +1,6 @@
+class AddBadgeStatusToProjects < ActiveRecord::Migration
+  def change
+    add_column :projects, :badge_status, :string
+    add_index :projects, :badge_status
+  end
+end
```

Comment:
You're going to need to add an additional step to the migration to add this value for items already in the database. This is untested, but you want to use [find_each](http://apidock.com/rails/ActiveRecord/Batches/find_each) to do something like:
```ruby
def up
  Project.find_each { |project| project.update_badge_status; project.save }
end

def down
end
```